package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@thundra/core",
-  "version": "2.12.3",
+  "version": "2.12.4",
```

Comment:
Bunu otomatik yapti, acaba bu degisikligi de pushlamama gerek var mi?

Reply:
@aoozdemir it is not needed. It is automatically increased while releasing

Reply:
@serkan-ozal Ah, I see what happened. `2.14.4` is already released but the lock file hasn't been updated. So, I guess it's better I keep this change, right?

Reply:
Yes, seems that synchronization between package.json and package-lock.json was somehow broken by a manual change. So it is OK to change it manually `2.14.4` here